spec/connect/migration_spec.rb

```diff
@@ -17,33 +17,42 @@
     let(:config) { SUSE::Connect::Config.new }
     let(:client) { SUSE::Connect::Client.new(config) }
     let(:status) { SUSE::Connect::Status.new(config) }
+    let(:product_tree) { JSON.parse(File.read('spec/fixtures/product_tree_migration.json')) }
+    let(:product) { Remote::Product.new(product_tree) }
+
     let(:service) do
-      SUSE::Connect::Remote::Service.new(name: 'SLES12', obsoleted_service_name: 'SLES11',
```

Comment:
Can we keep the SLE12 specs? We need it working on both, I would be good to have both scenarios covered with tests.

Reply:
Do you mean I should create two tests, yes for sure if you think that's the way to go :)